sophia/src/ns.rs

```diff
@@ -186,6 +200,44 @@ pub mod rdfs {
     );
 }
 
+/// The standard `xml:` namespace
+pub mod xml {
+    namespace!(
+        "http://www.w3.org/XML/1998/namespace#",
+        lang,
+        space,
+        base,
+        id,
+        // John Bosak
+        Father
+    );
+}
+
+/// The standard `dc:` namespace.
+pub mod dc {
```

Comment:
The `dc` prefix is ambiguous: it is sometimes used for `http://purl.org/dc/elements/1.1/`, and sometimes for `http://purl.org/dc/terms/`. In order to avoid any ambiguity, I prefer to use `dce` or `dc11` for the former, and `dct` for the later.

Reply:
I was using it for my tests, but ultimately stopped; since it is not used anywhere it could just be removed.

Reply:
works for me :)